test/integration/user-management-api/buyers_application_referrer_filters_test.rb

```diff
@@ -27,31 +29,31 @@ def setup
 
     get(admin_api_account_application_referrer_filters_path(@buyer, app))
     assert_response :forbidden
-    get(admin_api_account_application_referrer_filters_path(@buyer, app), access_token: token.value)
+    get(admin_api_account_application_referrer_filters_path(@buyer, app), params: { access_token: token.value })
     assert_response :not_found
     User.any_instance.expects(:member_permission_service_ids).returns([app.issuer.id]).at_least_once
-    get(admin_api_account_application_referrer_filters_path(@buyer, app), access_token: token.value)
+    get(admin_api_account_application_referrer_filters_path(@buyer, app), params: { access_token: token.value })
     assert_response :success
   end
 
   #TODO: clean the tests
   test 'create referrer filters' do
     application = @buyer.bought_cinstances.last
-    application_id = application.application_id
-    application.service.update_attribute :referrer_filters_required, true
+    application.application_id
```

Comment:
What is the point of this repeated statement? Not that I make any sense of the original.

Reply:
It makes no sense at all